weave/integrations/langchain/langchain.py

```diff
@@ -107,7 +101,7 @@ def _persist_run(self, run: Run) -> None:
         def _persist_run_single(self, run: Run) -> None:
             """Persist a run."""
             run_dict = _run_to_dict(run, as_input=True)
-            run_name = make_valid_run_name(run.name)
+            run_name = sanitize_call_name(run.name)
```

Comment:
The `<` case in `make_valid_run_name` is redundant with the regexp, but the `>` case is different - I'm guessing in this context `>` is at the end and a trailing `_` looks bad.